site/pages/gcweb-theme/static-header-footer/site-assets/x-bootstrap-4.min.css

```diff
@@ -0,0 +1,2 @@
+/* Custom style to mimic Canada.ca */
```

Comment:
Include the licence info + mention it is an experimental CSS and include the date of build